osu.Server.Spectator/Hubs/Metadata/MetadataHub.cs

```diff
@@ -65,9 +66,19 @@ public override async Task OnConnectedAsync()
                         versionHash = versionHash.Substring(versionHash.Length - 82, 32);
                 }
 
-                usage.Item = new MetadataClientState(Context.ConnectionId, Context.GetUserId(), versionHash);
+                usage.Item = new MetadataClientState(Context.ConnectionId, Context.GetUserId(), versionHash)
+                {
+                    UserStatus = UserStatus.Online
+                };
```

Comment:
This isn't going to work for the "appear offline" state. The initial broadcast of "online"-ness needs to not occur until the client updates its status via `UpdateStatus`. I'm already doing some work around this because it's broken in other ways, though. I think this PR can get away with just not having this here (in testing it still works fine).


```suggestion
                usage.Item = new MetadataClientState(Context.ConnectionId, Context.GetUserId(), versionHash);
```